ci-operator/step-registry/baremetal/lab/nfs-provisioner/baremetal-lab-nfs-provisioner-commands.sh

```diff
@@ -14,7 +14,7 @@ NFS_CLIENT_MIRRORED_IMAGE_URL="quay.io/openshifttest/nfs-subdir-external-provisi
 
 
 # Set the NFS_SERVER value to the same value as the AUX_HOST, unless explicitly set.
-NFS_SERVER=${NFS_SERVER:-${AUX_HOST}}
+NFS_SERVER=${NFS_SERVER:-$(<"${CLUSTER_PROFILE_DIR}/ntp-host")}
```

Comment:
```suggestion
NFS_SERVER=${NFS_SERVER:-$(<"${CLUSTER_PROFILE_DIR}/aux-host-internal-name")}
```

can we change any references to that `ntp-host` file to `aux-host-internal-name` to avoid confusion in the future when reading this?

Also fix the comment above

Reply:
Done